app/code/Magento/PageBuilder/view/adminhtml/web/js/form/element/validator-rules-mixin.js

```diff
@@ -67,7 +67,7 @@ define([
      * @return {Boolean}
      */
     function validateCssClass(str) {
-        return (/^[a-zA-Z _\-\d]+$/i).test(str);
+        return (/^[a-zA-Z\d\-_/:.\[\]&amp;@()! ]*$/i).test(str);
```

Comment:
I think `&amp;` here should be `&`. This isn't an HTML document, but a JavaScript file. `a` and `m` and `p` are already included in `a-z`, and I think `;` shouldn't be added as a valid class name character.

```suggestion
        return (/^[a-zA-Z\d\-_/:.\[\]&@()! ]*$/i).test(str);
```

Reply:
Thanks for the tip, you're absolutely right